src/main/golo/lazylist.golo

```diff
@@ -0,0 +1,355 @@
+# ............................................................................................... #
+#
+# Copyright (c) 2012-2015 Institut National des Sciences Appliquées de Lyon (INSA-Lyon)
```

Comment:
Note that once we are at Eclipse you will no longer be required to assign all copyright to INSA, since you will be a committer at Eclipse, and you will be just fine with your own copyright mention :-)